config/functions.cfg

```diff
@@ -211,18 +211,18 @@ function git_clone {
   #If OVERRIDE_CONFIGVER is set proceed to clone from that tag
   [[ ! -z "$OVERRIDE_CONFIGVER" ]] && CONFIGVER=$OVERRIDE_CONFIGVER
 
-  echo -e "${GREEN}Cloning into ${CYAN}elrond-config-$ENVIRONMENT${GREEN} with tag ${CYAN}$CONFIGVER${GREEN}...${NC}"
+  echo -e "${GREEN}Cloning into ${CYAN}mx-chain-$ENVIRONMENT-config${GREEN} with tag ${CYAN}$CONFIGVER${GREEN}...${NC}"
```

Comment:
I made migrate and then upgrade squad and got:

```
"Cloning into mx-chain-testnet-config with tag tags/T1.3.50.4...

fatal: destination path 'mx-chain-testnet-config' already exists and is not an empty directory."
```

Reply:
Added a check to make sure it exits properly